assets/helpers/csrf/csrfReducer.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Action } from './csrfActions';
+
+
+// ----- Types ----- //
+
+type Csrf = {
+    token: ?string,
+};
+
+
+// ----- Setup ----- //
+
+const initialState: Csrf = window.guardian && window.guardian.csrf ? {
```

Comment:
From my point of view, I guess that is better to keep this PR focus on CSRF and refactor UserReducer in a separate PR. Additionally, we can add tests to that reducer in that PR.